jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/NotificationUtils.java

```diff
@@ -160,7 +160,7 @@ public static boolean triggerAutoFixNotification(Map<String, String> policyParam
             }
         }
         catch(Exception exception){
-            LOGGER.error("exception occurred in triggerAutofixNotification with message - {}",exception.getMessage());
+            LOGGER.error("Error occurred in job triggerAutofixNotification with message - {}",exception.getMessage());
```

Comment:
consider extracting this to a constant and standardizing the errors per same comments as for the collectors